### PR TITLE
Fix FGVCAircraft dataset documentation

### PR DESCRIPTION
**Pull request by detkov**
There is a mistake in FGVCAircraft dataset documentation. If you load dataset and sum up 'trainval' and 'test' splits, there are not 10 200, but 10 000 images. Also mistake is in the number of classes: if we choose `annotation_level` to be `variant`, then unique number of classes is going to be not 102, but 100, and with `manufacturer` level it is not 41, but 30.

Initially was explored in issue #5809.

### CODE REVIEW

File: torchvision/datasets/fgvc_aircraft.py [hunk 1]
```diff
@@ -12,15 +12,15 @@
 class FGVCAircraft(VisionDataset):
     """`FGVC Aircraft <https://www.robots.ox.ac.uk/~vgg/data/fgvc-aircraft/>`_ Dataset.
 
-    The dataset contains 10,200 images of aircraft, with 100 images for each of 102
+    The dataset contains 10,000 images of aircraft, with 100 images for each of 100
```
Comment (pmeier) [inline torchvision/datasets/fgvc_aircraft.py:15]: Oh boy, they even got the number of samples wrong on their website? :roll_eyes: Thanks for checking and fixing!

**Review by pmeier — APPROVED** [review]
Thanks a ton @detkov. LGTM!

File: torchvision/datasets/fgvc_aircraft.py (hunk 1 above)
Comment (NicolasHug) [reply torchvision/datasets/fgvc_aircraft.py:15]: If the website is reporting the original values, are we doing something wrong in the code?

Comment (pmeier) [reply torchvision/datasets/fgvc_aircraft.py:15]: I don't think so, no:

```shell
$ cat variants.txt | wc -l
100
$ cat families.txt | wc -l
70
$ cat manufacturers.txt | wc -l
30
$ ls images | wc -l
10000

**Review by NicolasHug — APPROVED** [review]
Thanks @detkov and @pmeier

**Pull request merged**